dagfactory/dagfactory.py

```diff
@@ -106,9 +111,20 @@ def build_dags(self) -> Dict[str, DAG]:
                 dags[dag["dag_id"]]: DAG = dag["dag"]
             except Exception as err:
                 raise DagFactoryException(f"Failed to generate dag {dag_name}. verify config is correct") from err
+            else:
+                self.taskgroups_count = dag_builder.taskgroups_count
+                self.tasks_count = dag_builder.tasks_count
```

Comment:
Same question as above for the taskgroups count.

Reply:
Addressed in https://github.com/astronomer/dag-factory/pull/250#discussion_r1804325449